composer.json

```
@@ -7,11 +7,31 @@
 	"authors": [
 		{
 			"name": "Alley",
-			"homepage": "https://alley.co/"
+			"homepage": "https://alley.com/"
 		}
 	],
 	"support": {
 		"issues": "https://github.com/alleyinteractive/es-wp-query/issues",
 		"source": "https://github.com/alleyinteractive/es-wp-query"
-	}
+	},
+  "require-dev": {
+		"php": ">=7.4",
+		"alleyinteractive/alley-coding-standards": "2.0",
+		"yoast/phpunit-polyfills": "^1.0"
+	},
+  "scripts": {
+    "phpcbf": "phpcbf .",
+    "phpcs": "phpcs . --basepath=. --ignore=vendor/*",
```

Those values seem to be already present inside the `phpcs.xml.dist`.

```suggestion
    "phpcs": "phpcs",
```